Makefile

```diff
@@ -201,12 +204,33 @@ xgo:
 install-os-dependencies:
 	_assets/scripts/install_deps.sh
 
-setup: install-os-dependencies dep-install lint-install mock-install gen-install update-fleet-config ##@other Prepare project for first build
+setup: install-os-dependencies dep-install lint-install mock-install deploy-install gen-install update-fleet-config ##@other Prepare project for first build
 
 generate: ##@other Regenerate assets and other auto-generated stuff
 	go generate ./static ./static/migrations
 	$(shell cd ./services/shhext/chat && exec protoc --go_out=. ./*.proto)
 
+prepare-release: clean-release
+	mkdir -p $(RELEASE_DIRECTORY)
+	mv build/bin/statusgo-android-16.aar $(RELEASE_DIRECTORY)/status-go-android.aar
+	#mv build/bin/statusgo-ios-9.3-framework/status-go-ios.zip $(RELEASE_DIRECTORY)/status-go-ios.zip
+	${MAKE} clean
+	zip -r $(RELEASE_DIRECTORY)/status-go-desktop.zip . -x *.git*
+
+clean-release:
+	rm -rf $(RELEASE_DIRECTORY)
+
+release:
+	@read -p "Are you sure you want to create a new GitHub release? (y/n): " REPLY; \
+	if [ $$REPLY = "y" ]; then \
+	    github-release "status-im/status-go" "$(RELEASE_TAG)" "$(release_branch)" "" "$(RELEASE_DIRECTORY)/*"; \
```

Comment:
Should we validate `$(release_branch)` to make sure it is not empty (e.g. typo in running `make release`)?

Reply:
it would throw an error as it's a required parameter, but we decided to default it to `develop`